### PR TITLE
Add tasks to collect OVN/OVS logs

### PR DESCRIPTION
**Pull request by MosheSheena**
This change adds collection of OVN/OVS logs from both Ovirt Engine
and hypervisors
This is elementary in order to analyze failures in these components

### CODE REVIEW

**Comment by MosheSheena** [comment]
@kobihk Hi can you please review?

File: roles/ovirt-collect-logs/tasks/hypervisor.yml [hunk 1]
```diff
@@ -56,6 +56,21 @@
...
+    src: "{{ item.src }}"
+    dest: "{{ ovirt_collect_logs_tmp_dir }}/{{ item.dest }}"
+    state: link
+  with_items:
+    -
+      src: "/var/log/ovn"
+      dest: "ovn-logs"
+    -
+      src: "/var/log/openvswitch"
+      dest: "ovs-logs"
```
Comment (kobihk) [inline roles/ovirt-collect-logs/tasks/hypervisor.yml:70]: same here

**Review by kobihk — CHANGES_REQUESTED** [review]

File: roles/ovirt-collect-logs/tasks/engine.yml [hunk 2]
```diff
@@ -53,6 +53,11 @@
       src: "/var/log/openvswitch/ovsdb-server-nb.log"
       dest: "ovsdb-server-nb.log"
 
+    - src: "/var/log/ovn"
+      dest: "ovn-logs"
+        - src: "/var/log/openvswitch"
+          dest: "ovs-logs"
```
Comment (kobihk) [inline roles/ovirt-collect-logs/tasks/engine.yml:59]: please fix the indentation
```suggestion
    - src: "/var/log/openvswitch"
      dest: "ovs-logs"
```

File: roles/ovirt-collect-logs/tasks/hypervisor.yml [hunk 3]
```diff
@@ -54,6 +54,11 @@
       src: "/var/log/openvswitch/ovsdb-server.log"
       dest: "ovsdb-server.log"
 
+    - src: "/var/log/ovn"
+      dest: "ovn-logs"
+        - src: "/var/log/openvswitch"
+          dest: "ovs-logs"
```
Comment (kobihk) [inline roles/ovirt-collect-logs/tasks/hypervisor.yml:60]: same ^^

**Review by kobihk — CHANGES_REQUESTED** [review]

File: roles/ovirt-collect-logs/tasks/hypervisor.yml [hunk 4]
```diff
@@ -54,6 +54,12 @@
       src: "/var/log/openvswitch/ovsdb-server.log"
       dest: "ovsdb-server.log"
 
+    - src: "/var/log/ovn"
+      dest: "ovn-logs"
+
+    - src: "/var/log/openvswitch"
+      dest: "ovs-logs"
```
[the remark this answers is not in the document]
Comment (MosheSheena) [reply roles/ovirt-collect-logs/tasks/hypervisor.yml:61]: Same above.
The terms are interchangable

File: roles/ovirt-collect-logs/tasks/engine.yml [hunk 5]
```diff
@@ -53,6 +53,12 @@
       src: "/var/log/openvswitch/ovsdb-server-nb.log"
       dest: "ovsdb-server-nb.log"
 
+    - src: "/var/log/ovn"
+      dest: "ovn-logs"
+
+    - src: "/var/log/openvswitch"
+      dest: "ovs-logs"
```
[the remark this answers is not in the document]
Comment (kobihk) [reply roles/ovirt-collect-logs/tasks/engine.yml:60]: The question is if one will not overwrite the other one?

[the remark this answers is not in the document]
Comment (MosheSheena) [reply roles/ovirt-collect-logs/tasks/engine.yml:60]: Why should one overwrite the other?
On both Engine and hosts you collect logs to two distinct directories.

**Review by kobihk — APPROVED** [review]

**Comment by MosheSheena** [comment]
@kobihk Do you think CI is stuck?..
If so is there a way to re-trigger it without pushing any other commits?

File: roles/ovirt-collect-logs/tasks/engine.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (kobihk) [reply roles/ovirt-collect-logs/tasks/engine.yml]: yep you are right I missed it

**Review by kobihk — APPROVED** [review]

**Pull request merged**